skt/runner.py

```diff
@@ -87,12 +87,12 @@ def __load_blacklist(self, filepath):
                     if line:
                         hostnames.append(line)
         except (IOError, OSError) as exc:
-            logging.error('Can\'t access {}!'.format(filepath))
```

Comment:
Here as well, can you mention the specific warning in the commit message that is being solved?